L1Trigger/L1CaloTrigger/plugins/Phase1L1TJetProducer.cc

```diff
@@ -159,6 +159,18 @@ void Phase1L1TJetProducer::produce(edm::Event& iEvent, const edm::EventSetup& iS
     // dumping the data
     this -> _caloGrid -> Reset();
     this -> _fillCaloGrid<>(*(this -> _caloGrid), *inputCollectionHandle);
+
```

Comment:
Line 162 to 173 do not actually do anything, I would comment the entire block, not only the content of the double for-loop